test/performance/README.md

```diff
@@ -1,128 +1,168 @@
+   - 10 config maps
+
+   For Openshit clusters: 
+
+   ```bash
+   $ ./setup-clusterloader2.sh --env ocp
+   ```
+
+    In Openshift environments, a modified version of the kube-burner [cluster-density-ms](https://github.com/kube-burner/kube-burner-ocp/tree/478bb42e1842a94ca3210d26a08633b70a443005/cmd/config/cluster-density-ms) workload will be generated per namespace. This workload consists of the following objects:
+
+    - 1 image stream.
+    - 4 deployments, each with two pod replicas (pause), mounting 4 secrets, 4 config maps, and
+      1 downward API volume each.
+    - 2 services, each pointing to the TCP/8080 and TCP/8443 ports of the first and second
+      deployments, respectively.
+    - 1 edge route pointing to the first service.
+    - 20 secrets containing a 2048-character random string.
+    - 10 config maps containing a 2048-character random string
+
+
+3. Configure the parameters of your experiment:  
 
    a) cd into the load configuration directory
 
-```bash
-$ cd  perf-tests/clusterloader2/testing/load
-```
-   b) create the `stress-test` namespace - you workload needs to be deployed into this exact namespace (temporary solution)
-
-```bash
-$ kubectl apply -f - <<EOF
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: stress-test
-  labels:
-    app.kubernetes.io/cluster: wec1
-EOF
-```
-
-   c) configure your sample workload: 
-   
-```bash
-$ vi test-workload.yaml
-```
+   ```bash
+   $ cd  $WORK_DIR
+   ```
   
-   d) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
    
-```bash
-$ vi performance-test-config.yaml
-``` 
+   ```bash
+   $ vi performance-test-config.yaml
+   ``` 
+
+   More specifically, configure the following parameters: 
+
+   - namespaces: number of namespaced (*default value: 1*)
+   - namespaceNamePrefix: name prefix for the namespace (*default value: "perf-exp"*)
+   - K8S_CLUSTER: set to `true` for Kubernetes clusters (*default value: "true"*)
+   - OPENSHIFT_CLUSTER: set to `true` for Kubernetes clusters (*default value: "false"*)
+   - tuningSet: workload generation function (*default value: "RandomizedLoad"*)
+
+   To learn more about the clusterloader2 load experiments configuration parameters, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set).
+
+
+
 
-To learn more about the clusterloader2 load experiments configuration parameters, see the following: [clusterloader tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set)
+4. Run your first workload:
 
 
-3. Run your first workload:
+   Set ``--kubeconfig`` flag to the path of the kubeconfig file of your cluster (e.g., WDS kubeconfig) and use the newly added Kubestellar provider (`--provider=ks`)
```

Comment:
If this is a KubeStellar test then the user certainly wants to use a kubeconfig that accesses the WDS, right? That is not just an example.

What about the question of which _context_ in the kubeconfig to use? Does the `clusterloader.go` take a `--context` or some such flag? The user needs to be told what to do about making this script use the right kubeconfig context.

Reply:
Fixed. Added instructions on how to set the context to a KS WDS space.